pkg/vpcmodel/grouping.go

```diff
@@ -262,6 +273,42 @@ func (g *GroupConnLines) groupExternalAddressesForSubnets() {
 	g.GroupedLines = res
 }
 
+func (g *GroupConnLines) groupExternalAddressesForDiff(thisMinusOther bool) {
```

Comment:
There are now 3 functions that are very similar: `groupExternalAddressesForDiff` , `groupExternalAddresses`, `groupExternalAddressesForSubnets` -- all are about grouping external ip-blocks.. is it possible to generalize and have only one function?

Reply:
These function range over different content and do different operations on them. E.g., for `subnets` `src` can not be a public internet. Creating a unified datastructure for which we can have a single function that ranges over it will result with more code (and more functions) which is not readable, I'm afraid.
However, I will try to extract identical (or almost identical) code segments and reuse them.

Reply:
done

Reply:
Merged  2 of the 3; extracted reused code for all 3